### DISCUSSION TITLE
Win 3.6.0: Copying and pasting across resources, e.g. when using Deepl = not possible any more?

### DISCUSSION THREAD

**Discussion by moehzhaw** [SafeExamBrowser/seb-win-refactoring#763] [Q&A]
**The following situation applies:**
- During a test, additional resources are available, e.g. access to Deepl.
- After something has been translated, the translation should be copied from Deepl and then pasted into a free text task on Moodle.
- Status for Windwos: ❌does not work with my device (Windows 11) with the latest SEB Windows version (3.6.0); it still worked with Win version 3.5.0. (What still works under Win 3.6.0: I can still copy and paste text within the Moodle test... but not across resources).
- Status with macOS: ✅ Works with the latest Mac version (3.3)
- Conclusion: Due to this difference, there is an enormous inequality in the treatment of Windows and macOS users

**My questions:**
- Do other users have the same challenge?
- Is it possible that the latest Windows version (3.6.0) prevents copy-paste even more strictly?
- If the strict handling is not intended, what could be the reason why copy-paste no longer works?
- What workaround could solve the problem?

**Comment by dbuechel** [comment]
Please refer to #764 for further progress of this issue.
